eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -290,6 +291,10 @@ instance EraPParams era => FromCBOR (ConwayGovPredFailure era) where
 
 data ConwayGovEvent era
   = GovNewProposals !(TxId (EraCrypto era)) !(Proposals era)
+  | GovRemovedVotes
+      !(TxId (EraCrypto era))
+      !(Set (Voter (EraCrypto era), GovActionId (EraCrypto era)))
+      !(Set (Credential 'DRepRole (EraCrypto era)))
```

Comment:
Would be nice to have a haddock at least for this field, since it's perhaps not so intuitive what it represents.